frontend/src/project/ProjectContext.tsx

```diff
@@ -255,9 +215,10 @@ export const ProjectProvider: React.FC<Props> = (props) => {
   };
 
   const getRecommendedPairs = () => {
-    getRecommendedPairing(project.id).then((recommended) => {
-      setProject(recommended);
-    });
+    const recommendedConfiguration = recommendPairs(project, pairingHistory);
+    console.log("recommended", recommendedConfiguration);
```

Comment:
This might also be noise in console. Maybe we can remove when it's no longer needed for debugging?